src/components/datatable/DataTable.md

```diff
@@ -118,9 +166,49 @@ function Component() {
   const delimiters = { row: "\n", cell: "\t" };
 
   const options = {
+    pagination: pagination,
     page: 0,
     rowsPerPage: 10,
     rowsPerPageOptions: [10, 25, 50, 100],
+    onRowClick: (rowData) => {
+      const getRowData = rowData[1].props.tableMeta.rowData
+      const reference = getRowData[1].split(delimiters.cell)[0];
+      const [chapter, verse] = reference.split(":");
+      const getUid = rowData[2].props.tableMeta.rowData
+      const [uid] = getUid[2].split(delimiters.cell)[1];
+      let rowId = `header-${chapter}-${verse}-${uid}`;
+      setSaveRowId(rowId)
+    },
+
+    onSearchClose: () =>{
+      const testRefId = saveRowId;
+      const element = document.getElementById(testRefId);
+      if ( element ) {
+        element.scrollIntoView();
+      } else {
+        alert(`Element id not found: ${testRefId}`);
+        // setPagination(false);
+      }
+    },
+
+    onFilterChipClose: () =>{
+      console.log('dsddddd')
+      const testRefId = saveRowId;
+      const element = document.getElementById(testRefId);
+      if ( element ) {
+        element.scrollIntoView();
+      } else {
+        alert(`Element id not found: ${testRefId}`);
+      }
+    },
+    onFilterDialogOpen:() =>{
+      console.log('dsddddd')
+      setPagination(false)
+    },
+    onSearchOpen:() =>{
+      console.log('dsddddd')
```

Comment:
it would be helpful if it was at least different that the others. But really console.log should not  be left in production code.